include/internal/libspdm_fips_lib.h

```diff
@@ -0,0 +1,26 @@
+/**
+ *  Copyright Notice:
+ *  Copyright 2023 DMTF. All rights reserved.
+ *  License: BSD 3-Clause License. For full text see link: https://github.com/DMTF/libspdm/blob/main/LICENSE.md
+ **/
+
+#if LIBSPDM_FIPS_MODE
+
+/**
+ * HMAC KAT test: HMAC-SHA512 KAT and HMAC-SHA256 KAT.
```

Comment:
need add HMAC-SHA384

Reply:
Thanks. I have added the HMAC-SHA384 KAT.